packages/expo-video/src/__tests__/ExpoVideo-test.tsx

```diff
@@ -0,0 +1,54 @@
+import { render } from '@testing-library/react-native';
```

Comment:
Your unit tests are failing (see the check-packages test run result). I attempted to fix the import issue with the following change:

```diff
diff --git a/packages/expo-video/package.json b/packages/expo-video/package.json
index 3e7245201b..1324d40c85 100644
--- a/packages/expo-video/package.json
+++ b/packages/expo-video/package.json
@@ -31,8 +31,13 @@
   "license": "MIT",
   "dependencies": {},
   "devDependencies": {
+    "@types/jest": "^29.5.14",
+    "@testing-library/react-native": "^12.5.2",
     "expo-module-scripts": "^4.0.0"
   },
+  "jest": {
+    "preset": "expo-module-scripts"
+  },
   "peerDependencies": {
     "expo": "*",
     "react": "*",
```

However, the tests still fail with a different set of errors.